mobile/src/components/common/text/text.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { Text } from 'react-native';
+import { selectFontColor, selectFontUsage } from './helpers';
+
+type Props = {
+  innerText: string;
```

Comment:
We should accept here children and style. You need to exclude font-weight and font-style from style type and also make fontFamily AppFontFamily type
We have to accept children and style here. You need to exclude font-weight and font-style from style type and also make fontFamily equal to our enum